packages/camera/camera_android_camerax/lib/src/camerax_proxy.dart

```diff
@@ -151,7 +153,7 @@ class CameraXProxy {
   Camera2CameraControl Function(CameraControl cameraControl)
       getCamera2CameraControl;
 
-  /// Create [CapureRequestOptions] with specified options.
+  /// Create [CaptureRequestOptions] with specified options.
```

Comment:
extreme nit but (for consistency)
```suggestion
  /// Creates a [CaptureRequestOptions] with specified options.
```